csrankings-b.csv

```diff
@@ -803,10 +803,10 @@ Brygg Ullmer,Louisiana State University,https://www.lsu.edu/eng/cse/people/facul
 Bu-Sung Lee,Nanyang Technological University,http://research.ntu.edu.sg/expertise/academicprofile/Pages/StaffProfile.aspx?ST_EMAILID=ebslee,TyytTfQAAAAJ
 Bud Mishra,New York University,https://cs.nyu.edu/mishra,kXVBr20AAAAJ
 Budi Arief,University of Kent,https://www.cs.kent.ac.uk/people/staff/ba284,bSZk250AAAAJ
-Bugra Gedik,Bilkent University,http://www.cs.bilkent.edu.tr/~bgedik/homepage/doku.php,ddLcpMQAAAAJ
 Bundit Laekhanukit,SUFE,http://cs.mcgill.ca/~blaekh,vmemchMAAAAJ
 Buntarou Shizuki,University of Tsukuba,https://www.iplab.cs.tsukuba.ac.jp/~shizuki/index-e.html,lXG_jWEAAAAJ
-Burcu Can,Hacettepe University,https://web.cs.hacettepe.edu.tr/~burcucan,BalzcSEAAAAJ
+Burak Kurkcu,Hacettepe University,http://web.cs.hacettepe.edu.tr/~bkurkcu,IRcDIOwAAAAJ
+Burkay Genç,Hacettepe University,http://burkay.tk,w-pGWs4AAAAJ

```

Comment:
Appointment not in CS (per CV, "Inst. of Population Studies; Policy and Strategy Studies"). Please remove or clarify why they meet the inclusion criteria.

Reply:
New members are officially listed here -> https://www.cs.hacettepe.edu.tr/#faculty_members